src/synonym_map.c

```diff
@@ -92,10 +92,12 @@ void SynonymMap_Free(SynonymMap* smap) {
   rm_free(smap);
 }
 
+char *strtolower(char *str); // in spec.c
```

Comment:
put in common.h

Reply:
Sorry common.h is not a good place, we need to have some common file to have all those functions. but till then why not put it in spec.h and include it?

Reply:
Is it OK that I'm changing the original RedisString or should I duplicate?